storage-resize-images/extension.yaml

```diff
@@ -114,6 +114,20 @@ params:
       Invalid sizes, must be a comma-separated list of WIDTHxHEIGHT values.
     required: true
 
+  - param: DELETE_ORIGINAL_FILE
+      type: select
+      label: Delete Original File? (Yes/No)
+      description: >-
+        Delete the original file from the Cloud Storage bucket? This may be useful if you wish to store
```

Comment:
Hi! Thanks so much for this PR! I think it might be good to add some text here to warn users that they cannot undo this delete. Adding our tech writer for her review.

Reply:
@karayu Thanks! Added the deletion warning. Waiting for Rachel's review.

Reply:
Made edits directly in the file in this PR.